optimum/intel/ipex/modeling_base.py

```diff
@@ -169,9 +176,17 @@ def __init__(
             self._init_warmup()
 
     @classmethod
-    def _from_transformers(
+    def _from_transformers(cls, *args, **kwargs):
+        logger.warning(
+            "_from_transformers will be removed in the future, please remove the export arguments in the API."
+        )
```

Comment:
I think we should remove this warning as  user still should load the model with :  `.from_pretrained(.., export=True)` 
```suggestion
```